src/Services/ConvertToLearnosityService.php

```diff
@@ -140,9 +144,26 @@ public function process()
         FileSystemHelper::createDirIfNotExists($this->outputPath . DIRECTORY_SEPARATOR . $this->rawPath);
 
         $this->assetsFixer = new AssetsFixer($this->organisationId);
-
-        $result = $this->parseContentPackage();
-
+        if ($this->isSingleItemConvert == 'Y' || $this->isSingleItemConvert == 'YES') {
+            $finder = new Finder();
+            $finder->files()->in($this->inputPath);
```

Comment:
@rakeshqdsa the idea was that you pass a direct file system path to a single file, not a directory that might contain a file.

So, I might pass this:

```
./bin/mo convert:to:learnosity --input ~/Desktop/my-qti.xml --output ~/Desktop/_output --single-item yes
```

Reply:
Done